mcs/resource_mcs_db_cluster.go

```diff
@@ -140,7 +140,7 @@ func resourceDatabaseCluster() *schema.Resource {
 							ForceNew: true,
 							ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
 								v := val.(string)
-								if v != Galera && v != Postgres {
+								if v != Galera && v != Postgres && v != Tarantool {
```

Comment:
maybe use something like this:

`
if availableClusterDatastores(v){


func availableClusterDatastores(ds string) bool {
	for _, clusterDatastore := range getClusterDatastores() {
		if ds == clusterDatastore {
			return true
		}
	}
	return false
}`

Reply:
so there will be no need to change the code in 2 places